magnet_derive/src/meta.rs

```diff
@@ -4,6 +4,21 @@ use std::f64;
 use syn::{ Attribute, Meta, NestedMeta, MetaNameValue, Lit };
 use error::{ Error, Result };
 
+pub fn doc_meta(attrs: &[Attribute]) -> Option<MetaNameValue> {
```

Comment:
Please add doc comments to all items (especially public ones). Running `cargo +nightly clippy` can show you wherever they are missing.